high_performance/modules/high_performance/include/klepsydra/high_performance/event_loop_scheduler.h

```diff
@@ -46,8 +46,12 @@ class EventLoopScheduler : public Scheduler
     void startScheduledTask(const std::string &name,
                             int after,
                             bool repeat,
-                            std::function<void()> function) override;
-    void startScheduledService(int after, bool repeat, Service *service) override;
+                            std::function<void()> task,
+                            const std::vector<int> &cpuAffinity = {}) override;
```

Comment:
remove affinity

Reply:
Removing from [kpst-sdk](https://github.com/klepsydra-technologies/kpsr-sdk/blob/c5d43384bd9a08e5b0501072feb796135a0b91e0/modules/sdk/include/klepsydra/sdk/scheduler.h) too, @mharshe?

Reply:
yes remove from kpsr-sdk. Didn't realize this before there.